charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -82,7 +82,6 @@ data:
       auth:
         keys:
           - secret: {{ printf "${%s}" .Values.rhdhOperator.secretRef.backstage.backendSecret }}
-
```

Comment:
this change will attempt to release a new version and will fail since the version isn't bumped.
I'm going to merge it, so pls ignore the failure of the github action.